tdms/src/iterator.cpp

```diff
@@ -4873,86 +4863,73 @@ void extractPhasorsSurface(double **surface_EHr, double **surface_EHi, ElectricS
 
-      //now update the master array
-      surface_EHr[0][vindex] = surface_EHr[0][vindex] + real(subResultE);
-      surface_EHi[0][vindex] = surface_EHi[0][vindex] + imag(subResultE);
+        //now update the master array
+        surface_EHr[0][vindex] = surface_EHr[0][vindex] + real(subResultE);
+        surface_EHi[0][vindex] = surface_EHi[0][vindex] + imag(subResultE);
 
-      surface_EHr[3][vindex] = surface_EHr[3][vindex] + real(subResultH);
-      surface_EHi[3][vindex] = surface_EHi[3][vindex] + imag(subResultH);
+        surface_EHr[3][vindex] = surface_EHr[3][vindex] + real(subResultH);
+        surface_EHi[3][vindex] = surface_EHi[3][vindex] + imag(subResultH);
 
-      /*Ey and Hy*/
-      subResultH = Hy * cphaseTermH;//exp(phaseTermH * IMAGINARY_UNIT) * 1./((double) Nt);
-      subResultE = Ey * cphaseTermE;//exp(phaseTermE * IMAGINARY_UNIT) * 1./((double) Nt);
+        /*Ey and Hy*/
+        subResultH = Hy * cphaseTermH;//exp(phaseTermH * IMAGINARY_UNIT) * 1./((double) Nt);
+        subResultE = Ey * cphaseTermE;//exp(phaseTermE * IMAGINARY_UNIT) * 1./((double) Nt);
 
-      //now update the master array
-      surface_EHr[1][vindex] = surface_EHr[1][vindex] + real(subResultE);
-      surface_EHi[1][vindex] = surface_EHi[1][vindex] + imag(subResultE);
+        //now update the master array
+        surface_EHr[1][vindex] = surface_EHr[1][vindex] + real(subResultE);
+        surface_EHi[1][vindex] = surface_EHi[1][vindex] + imag(subResultE);
 
-      surface_EHr[4][vindex] = surface_EHr[4][vindex] + real(subResultH);
-      surface_EHi[4][vindex] = surface_EHi[4][vindex] + imag(subResultH);
+        surface_EHr[4][vindex] = surface_EHr[4][vindex] + real(subResultH);
+        surface_EHi[4][vindex] = surface_EHi[4][vindex] + imag(subResultH);
 
 
-      /*Ez and Hz*/
-      subResultH = Hz * cphaseTermH;//exp(phaseTermH * IMAGINARY_UNIT) * 1./((double) Nt);
-      subResultE = Ez * cphaseTermE;//exp(phaseTermE * IMAGINARY_UNIT) * 1./((double) Nt);
+        /*Ez and Hz*/
+        subResultH = Hz * cphaseTermH;//exp(phaseTermH * IMAGINARY_UNIT) * 1./((double) Nt);
+        subResultE = Ez * cphaseTermE;//exp(phaseTermE * IMAGINARY_UNIT) * 1./((double) Nt);
 
-      //now update the master array
-      surface_EHr[2][vindex] = surface_EHr[2][vindex] + real(subResultE);
-      surface_EHi[2][vindex] = surface_EHi[2][vindex] + imag(subResultE);
+        //now update the master array
+        surface_EHr[2][vindex] = surface_EHr[2][vindex] + real(subResultE);
+        surface_EHi[2][vindex] = surface_EHi[2][vindex] + imag(subResultE);
 
-      surface_EHr[5][vindex] = surface_EHr[5][vindex] + real(subResultH);
-      surface_EHi[5][vindex] = surface_EHi[5][vindex] + imag(subResultH);
-    }
+        surface_EHr[5][vindex] = surface_EHr[5][vindex] + real(subResultH);
+        surface_EHi[5][vindex] = surface_EHi[5][vindex] + imag(subResultH);
+      }

```

Comment:
I miss why the indentation changed here.

Reply:
It shouldn't have - probably an artefact of me untangling some of the interpolation logic above. It's now indented properly.